core/utils.py

```diff
@@ -232,6 +232,22 @@ def dict_from_yaml(yaml_str: str) -> Dict[str, Any]:
         raise InvalidInputException(e)
 
 
+def yaml_from_dict(dictionary: Dict[str, Any], width: int =80) -> Any:
```

Comment:
```suggestion
def yaml_from_dict(dictionary: Dict[str, Any], width: int = 80) -> str:
```

Reply:
Any was returned because mypy was throwing this error 
core/utils.py:246: error: Returning Any from function declared to return "str"  [no-any-return]

Reply:
rest done